SPECS/xcp-ng-xapi-storage.spec

```diff
@@ -47,7 +47,7 @@ Requires:       xcp-ng-xapi-storage-datapath-tapdisk
 Requires:       zfs >= 2.1
 
 %description    volume-zfsvol
-SMAPIv3 Volume plugins storing each VDI in a ZFS volume.
+SMAPIv3 Volume plugin managing ZFS volumes.
```

Comment:
There is a single plugin, but otherwise I find my original formulation to be more clear.  I also don't understand the part of the commit message describing the change.

Reply:
We just don't store any VDI.

Reply:
Well, to me it is still a raw VDI, it is just not a VHD- or QCOW2-formatted one.  What I intended to convey is, even though strictly speaking "VDI" (and "PBD") is a XAPI-level concept, and SMAPIv3 matches to a "Volume" concept, what the XAPI/XO user gets is a VDI that gets stored as a zfs volume.

Reply:
The issue I see is that people that are using zfs smapi v1 are used to store VDI file (not in the sense of the XAPI VDI object) into the zpool and so in the description saying that the plugin is storing VDI seems confusing to me but ok we can keep it no worries.

Reply:
In fact I just looked on my system and files are vhd. I don't know why but in my mind the extension was .vdi... Ok my bad.